### PR TITLE
Adding WPCourses education card to My Home

### PR DESCRIPTION
**Pull request by donlair**
This PR adds a WP Courses card to the Learn and Grow section of My Home as discussed at pcbrnV-264-p2.

Here's a screenshot of the card in place:

Screen Shot 2021-04-20 at 3 19 26 PM

The card is being added to these views:
* VIEW_DEFAULT
* VIEW_CLOUDFLARE
* VIEW_EARN_FEATURES
* VIEW_FIND_DOMAIN
* VIEW_CONNECT_ACCOUNTS
* VIEW_WEBINARS
* VIEW_PODCASTING

#### Testing instructions
* Patch your sandbox with D60187-code
* Update your hosts file to point public-api.wordpress.com to your sandbox
* Checkout this branch and start Calypso
* Force views for each view listed above but loading: http://calypso.localhost:3000/home/SITESLUG?dev=true&view=VIEW_CLOUDFLARE
* Click the link in the card and confirm that it behaves as expected.

### CODE REVIEW

File: client/my-sites/customer-home/locations/secondary/learn-grow/index.jsx [hunk 1]
```diff
@@ -42,7 +45,7 @@ const LearnGrow = ( { cards, trackCards } ) => {
 	if ( ! cards || ! cards.length ) {
 		return null;
 	}
-
+	console.dir( cardComponents );
```
Comment (blackjackkent) [inline client/my-sites/customer-home/locations/secondary/learn-grow/index.jsx]: Did you mean to leave this console statement in here?

**Review by blackjackkent — COMMENTED** [review]
This looks good to me and tests well! Left one comment about what seems to be a leftover console log, but other than that this is good to go.

File: client/my-sites/customer-home/locations/secondary/learn-grow/index.jsx (hunk 1 above)
Comment (donlair) [reply client/my-sites/customer-home/locations/secondary/learn-grow/index.jsx]: Nope, I removed it!

**Comment by donlair** [comment]
@blackjackkent Thanks Ros, I removed the console log and added an EN check since the course is available only in English!

**Review by blackjackkent — APPROVED** [review]

**Pull request merged**